rgw/v2/tests/nfs_ganesha/initialize.py

```diff
@@ -92,7 +94,7 @@ def do_un_mount(self):
         log.info("un_mounting dir: %s" % self.rgw_user_info["nfs_mnt_point"])
         un_mount_cmd = "sudo umount %s" % self.rgw_user_info["nfs_mnt_point"]
         un_mounted = utils.exec_shell_cmd(un_mount_cmd)
-        if un_mounted:
+        if un_mounted != "":
```

Comment:
add TODO comment here also which you have added [here](https://github.com/red-hat-storage/ceph-qe-scripts/pull/240/files#diff-d13c2a01592688685d9f1a566d9c7437750ef3ca59d00224cfc40feaca805519R166-R167)

Reply:
sure added